install.sh

```diff
@@ -270,20 +298,13 @@ EOS
     ln -srf "$retc"/systemd/system/node-exporter.service.d/nonroot.conf ~/.config/systemd/user/node-exporter.service.d
 fi
 
-install -m755 scylla-gdb.py -Dt "$rprefix"/scripts/
+install -m644 scylla-gdb.py -Dt "$rprefix"/scripts/
 
-PYSCRIPTS=$(find dist/common/scripts/ -maxdepth 1 -type f -exec grep -Pls '\A#!/usr/bin/env python3' {} +)
+PYSCRIPTS=$(find "$rprefix"/scripts/ -maxdepth 1 -type f -executable -exec grep -Pls '\A#!/usr/bin/env python3' {} +)
```

Comment:
These changes are good, but are unrelated

Reply:
Dropped these changes for now, adjust relocate_python3 function to do same behavior with relocate_python_script.py